packages/ui/src/components/CallInfo.tsx

```diff
@@ -140,11 +211,11 @@ const CallInfo = ({
           annoyance.
         </AlertStyled>
       )}
-      {args && Object.keys(args).length > 0 && (
+      {args && !!api && Object.keys(args).length > 0 && (
```

Comment:
I really like it when we encapsulate such logic in the method. There is no super complex logic, but it's still hard to read. What about making the method:
```js
const containsArgs = () => args && Object.keys(args).length > 0

{containsArgs() && !!api (<div></div>)}
```

Also the variable casting for "api" is not 100 percent right as it could be an empty object which is returns true, it doesn't give us confidence that API is initialized. 

I personally use lodash for such scenarios to not create new methods or check for the emptiness of the objects (https://lodash.com/docs/4.17.15#isEmpty)

Reply:
Lodash is hiding things behind a giant lib that has had many vulns in the past, it's a strong no for me. These are tastes after all. I don't like to hide things too much when they're a 1 line, and if they're not reused.. I've been involved in such projects and it's easy to read, but hard to maintain. I'll implement this suggestion, but I don't want this kind of comment to be a requirement for this repo, or to prevent merging after being up for 6 days.